src/praisonai-deploy/praisonai_deploy/cli/features/deploy.py

```
@@ -545,8 +648,18 @@ def handle_deploy_command(args):
         '--registry': 'registry',
         '--service-name': 'service_name',
         '--subscription-id': 'subscription_id',
+        '--stack-dir': 'stack_dir',
+        '--template': 'template',
+        '--dir': 'directory', '-d': 'directory',
+        '--chart': 'chart',
+        '--release': 'release', '-r': 'release',
```

Ruff (0.16.0)

[error] Dictionary key literal `'-r'` repeated

(F601)